src/Widgets/HeadlineWidget/HeadlineWidgetClass.js

```diff
@@ -2,7 +2,7 @@ import * as Scrivito from "scrivito";
 
 export const HeadlineWidget = Scrivito.provideWidgetClass("HeadlineWidget", {
   attributes: {
-    headline: "string",
+    headline: "html",
```

Comment:
* [ ] If we do this, we must update the `id={speakingUrl(widget.get("headline"))}` part.